napi_memory_management.md

```diff
@@ -0,0 +1,143 @@
+# Memory Management
+
+As N-API calls are made, references to objects in the heap for the underlying
+vm may be returned as ```napi_value```.  These references must hold the
```

Comment:
Is there a reason you're using triple backticks for all inline code? Single backticks should be sufficient for formatting inline code; triple backticks are only necessary for fencing code blocks, as far as I know.